docs/conditional.rst

```diff
@@ -142,7 +122,7 @@ You can check for conditional operation failures by inspecting the cause of the
 .. code-block:: python
 
     try:
-        thread_item.save(forum_name__null=False)
+        thread_item.save(Thread.forum_name.exists())
     except PutError as e:
```

Comment:
This doesn't have to be in this PR. But this kind of sucks. Can't we throw a generic ConditionalError that is subclassed from a PutError that people can catch? I imagine people would want to do conditional writes quite often to save on reads.